<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KUserFeedback::Console::SchemaEditWidget</class>
 <widget class="QWidget" name="KUserFeedback::Console::SchemaEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="schemaView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected element or entry.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAddSource">
   <property name="icon">
    <iconset theme="folder-add"/>
   </property>
   <property name="text">
    <string>Add &amp;Source...</string>
   </property>
   <property name="toolTip">
    <string>Add a new data source.</string>
   </property>
  </action>
  <action name="actionAddSourceElement">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Add Source &amp;Element...</string>
   </property>
   <property name="toolTip">
    <string>Add an element to a data source.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
